### ISSUE TITLE
Status bar won't update when Soft Tabs is toggled from Command Palette

### ISSUE THREAD

**Issue by hatzopoulos** [lee-dohm/indentation-indicator#10]
If I toggle my soft tabs via command palette, `Editor: Toggle Soft Tabs`, the status bar doesn't update.

**Comment by lee-dohm** [comment]
I think this is essentially another consequence of atom/atom#3719, but I'll double-check.

**Comment by lexicalunit** [comment]
Not sure if this is the same issue or not but going into the JS console and executing something like

```js
atom.workspace.getActiveTextEditor().setTabLength(N)
```

Also does not cause the indicator to update with the new tab length.

**Comment by lee-dohm** [comment]
Yes, unfortunately there isn't an event that will allow us to receive a notification of when indentation-relevant settings have changed. I could potentially listen to noisier events like `onDidStopChanging`, but I feel that the workaround of changing to another tab and back is about the same.
